Project/Gem/CMakeLists.txt

```diff
@@ -57,12 +57,19 @@ ly_add_target(
         PUBLIC
```

Comment:
Could you please add explanation of the purpose of the shape publisher, as well as documentation (README), if applicable?

Reply:
I added a short explanation in the header file.